unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWEntityHandlerShould.cs

```diff
@@ -83,13 +90,195 @@ public void EntityDuplicateName()
+
+        //Assert
+        Assert.IsFalse(entityHandler.IsAnyEntitySelected());
+    }
+
+    [Test]
+    public void ChangeLockStateSelectedEntities()
+    {
+        //Arrange
+        entity.IsLocked = false;
+        entityHandler.SelectEntity(entity);
+
+        //Act
+        entityHandler.ChangeLockStateSelectedEntities();
+
+        //Assert
+        Assert.IsTrue(entity.IsLocked);
+    }
+
+    [Test]
+    public void ChangeShowStateSelectedEntities()
+    {
+        //Arrange
+        entityHandler.SelectEntity(entity);
+        entity.IsVisible = true;
+
+        //Act
+        entityHandler.ChangeShowStateSelectedEntities();
+
+        //Assert
+        Assert.IsFalse(entity.IsVisible);
+    }
+
+    [Test]
+    public void ShowAllEntities()
+    {
+        //Arrange
+        entity.IsVisible = false;
+
+        //Act
+        entityHandler.ShowAllEntities();
+
+        //Assert
+        Assert.IsTrue(entity.IsVisible);
+    }
+
+    [Test]
+    public void GetAllVoxelEntities()
+    {
+        //Arrange
+        entity.isVoxel = true;
+
+        //Act
+        var voxelsEntities = entityHandler.GetAllVoxelsEntities();
+
+        //Assert
+        Assert.Greater(voxelsEntities.Count, 0);
+    }
+
+    [Test]
+    public void DesotryLastCreatedEntities()
+    {
+        //Arrange
+        var newEntity = entityHandler.CreateEmptyEntity(scene, Vector3.zero, Vector3.zero, false);
+        newEntity.IsNew = true;
+        entityHandler.SelectEntity(newEntity);
+        //Act
+        entityHandler.DestroyLastCreatedEntities();
+
+        //Assert
+        Assert.IsTrue(newEntity.IsDeleted);
+    }
+
+    [Test]
+    public void DeleteSingleEntity()
+    {
+        //Arrange
+        var newEntity = entityHandler.CreateEmptyEntity(scene, Vector3.zero, Vector3.zero, false);
+        newEntity.IsNew = true;
+
+        //Act
+        entityHandler.DeleteSingleEntity(newEntity);
+
+        //Assert
+        Assert.IsTrue(newEntity.IsDeleted);
+    }
+
+    [UnityTest]
+    public IEnumerator DeleteEntitiesOutsideBoundaries()
+    {
+        //Arrange
+        var newEntity = entityHandler.CreateEmptyEntity(scene, Vector3.one * 99999, Vector3.one * 99999, false);
+        newEntity.IsNew = true;
+        newEntity.rootEntity.transform.position = Vector3.one * 444;
+
+        TestHelpers.CreateAndSetShape(scene, newEntity.rootEntity.entityId, DCL.Models.CLASS_ID.GLTF_SHAPE, JsonConvert.SerializeObject(
+            new
+            {
+                src = TestAssetsUtils.GetPath() + "/GLB/Trunk/Trunk.glb"
+            }));
+
+        LoadWrapper gltfShape = GLTFShape.GetLoaderForEntity(scene.entities[newEntity.rootEntity.entityId]);
+        yield return new DCL.WaitUntil(() => gltfShape.alreadyLoaded);
+
+        //Act
+        entityHandler.DeleteEntitiesOutsideSceneBoundaries();
+
+        //Assert
+        Assert.IsTrue(newEntity.IsDeleted);
+    }
+
+    [Test]
+    public void AreEntitiesInsideBoundaries()
```

Comment:
```suggestion
    public void AreEntitiesInsideBoundaries()
```
We should rename this test. As its read as "biw entity handler should are entities inside boundaries"